hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/AlertsEngineImpl.java

```diff
@@ -214,11 +214,11 @@ public void reloadTrigger(final String tenantId, final String triggerId) {
         try {
             trigger = definitions.getTrigger(tenantId, triggerId);
         } catch (Exception e) {
-            log.debugf(e.getMessage(), e);
+            log.debug(e.getMessage(), e);
             msgLog.errorDefinitionsService("Trigger", e.getMessage());
         }
         if (null == trigger) {
-            log.debugf("Trigger not found for triggerId [" + triggerId + "], removing from rulebase if it exists");
+            log.debug("Trigger not found for triggerId [" + triggerId + "], removing from rulebase if it exists");
```

Comment:
we should use if ( log.debugEnabled() ) { log.debug(...); } whenthe debug logging is not of a literal string and not in  exception handling.  So, in several places in these changes.

As it turns out, this was already a poorly written log.debugf(), as it used string concat and not variable args...